Src/WitsmlExplorer.Frontend/components/ContentViews/CurveValuesView.tsx

```diff
@@ -63,18 +65,20 @@ export const CurveValuesView = (): React.ReactElement => {
       const startIndex = getCurrentMaxIndex();
       const endIndex = getOffsetIndex(startIndex, TIME_INDEX_OFFSET, DEPTH_INDEX_OFFSET);
       getLogData(startIndex, endIndex).then(() => {
-        timer.current = setTimeout(() => setRefreshFlag((flag) => !flag), refreshDelay * MILLIS_IN_SECOND);
+        refreshDelayTimer.current = setTimeout(() => setRefreshFlag((flag) => !flag), refreshDelay * MILLIS_IN_SECOND);
       });
     }
   }, [refreshFlag]);
 
   useEffect(() => {
     if (autoRefresh) {
       setRefreshFlag((flag) => !flag);
+      stopAutoRefreshTimer.current = setTimeout(() => setAutoRefresh(false), AUTO_REFRESH_TIMEOUT * MILLIS_IN_SECOND * SECONDS_IN_MINUTE); // Stop auto refresh after 5 minutes to reduce load on the server
```

Comment:
I suggest notifying the user that streaming has been automatically turned off, as it can be difficult to notice that only the stream button has been toggled off.

Reply:
Added a popup. 

About the other question: When turning on the stream, it always fetches the newest data for the log. If the selected interval does not cover the end of the log, it would think the fetched data is new even though it was just not within the selected interval. So it could make people think new data was recently added even if it was not.

Reply:
If the rest of the team has discussed and approved this behavior, I am okay with the decision.

Reply:
That it fetches the newest data before streaming has been discussed. We have not discussed that the first element is not green, but it doesn't really make sense to me that data rows added before the stream is enabled should be green.